cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -461,3 +462,44 @@ def test_reports_online_vms(appliance, setup_provider, provider, request, vm):
 
     view = navigate_to(saved_report, "Details")
     assert vm not in [row.vm_name.text for row in view.table.rows()]
+
+
+@pytest.mark.tier(0)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.uncollectif(
+    lambda case_sensitive: not case_sensitive
+    and BZ("1741588", forced_streams=["5.11"]).blocks
```

Comment:
This BZ is marked for 5.11 , no need of forced_stream .

Reply:
Oh yes, it wasn't triaged at the time. Removing it.

Reply:
I checked, but it's not working as expected. I think I'll need to keep it as is.